include/util/edit.h

```diff
@@ -64,6 +63,9 @@ class EditEngine {
   };
 
   ui::FileBlobModel* original_data_;
+  // TODO(catsuryuu): change to std::map after switching to C++17 (and use
+  // `insert_or_assign`)
+  // Reason: QMap works on `int` type
```

Comment:
'QMap works on `int` type' -> 'QMap uses `int` for container size'
Current version may be unclear for a reader, especially that in the next line you use `size_t`.

Reply:
Fixed